### ISSUE TITLE
About comparison between Mikado and EvidentialGene

### ISSUE THREAD

**Issue by juntaosdu** [EI-CoreBioinformatics/mikado#298]
Hi，
As I know, the EvidentialGene pipeline only takes the FASTA format file as input. So how do you compare the performance of Mikado and EvidentialGene?

Thank you!

**Comment by lucventurini** [comment]
Dear @juntaosdu 

We go into details about this in our paper of Mikado -[doi:10.1093/gigascience/giy093](https://academic.oup.com/gigascience/article/7/8/giy093/5057872). 

Briefly, after performing EviGene on the samples, we did align them against the genome of reference (with GMAP, if memory serves - details in the paper) and compared the structures against the reference annotation.

**Comment by juntaosdu** [comment]
Hi,
I have carefully searched your paper, but found nothing about this. Could you please help with it?

Thank you!

**Comment by lucventurini** [comment]
> Hi,
> I have carefully searched your paper, but found nothing about this. Could you please help with it?
> 
> Thank you!

Hi @juntaosdu , many apologies, we really should have explained this better in the paper. My memory was also wrong.

In short, what we did was:
1. combine all assemblies into a single GTF (no redundancy removal etc)
2. extract the sequences of all candidate transcripts
3. run EvidentialGene
4. extract from the input GTF the transcripts that were selected as "best" (ie the okayset) by EvidentialGene.

The snakefile can be found here: https://github.com/lucventurini/mikado-analysis/blob/master/Snakemakes/external.snakefile

In general the repository contains all the information about how we did the analyses for the paper.

I hope this helps.
